array/DNAm/preprocessing/jobSubmission/1_runDNAmQC.sh

```diff
@@ -25,12 +25,18 @@ echo Log files intially stored in: ${SLURM_SUBMIT_DIR}/QCDNAdata_${SLURM_JOB_ID}
 
 source $1 || exit 1
 
+
+if [[ -z "${DNAM_CONDA_ENVIRONMENT}" ]]; then
+    echo "Conda environment does not exist, please run the setup script first."
+    echo "The setup script can be found at ${SCRIPTSDIR}/array/DNAm/Setup/setup.sh"
+    exit 1
+fi
+    
+
 echo "Processing data located in :" ${DATADIR}
 
-## load modules
-echo "Loading R module :" $RVERS
-module load Pandoc
-module load $RVERS   # load specified R version
+source "${CONDA_SHELL}"
```

Comment:
Are these new shell variables that need to be included in the config template?

Reply:
They don't need to be added to the config template. To make things easier for the user the setup script just adds these values for you.

Reply:
I haven't removed `RVERS` from the example config file just yet as it is still being used in the other pipelines.